tools/ci_build/github/azure-pipelines/orttraining-linux-nightly-ortmodule-test-pipeline.yml

```diff
@@ -13,6 +13,11 @@ jobs:
 
   # Entry point for all ortmodule training tests
   - script: |
+      COMMIT_ID=$(python3 -c "import onnxruntime; print(onnxruntime.get_build_info().split('git-commit-id=')[1].split(',')[0])")
+      cd $(Build.SourcesDirectory)
+      git checkout $COMMIT_ID
+      git branch
```

Comment:
Why do we need `git branch`?

Reply:
logging purpose.
if it fails in the future, it will be easy to debug.